specification.md

```diff
@@ -1565,6 +1569,56 @@ For more information about functions, reference the [Functions definitions](#Fun
 
 For more information about workflow expressions, reference the [Workflow Expressions](#Workflow-Expressions) section.
 
+#### Defining custom function types
+
+[Function definitions](#function-definition) `type` property defines a list of function types that are set by
+the specification.
+
+Some runtime implementations might support additional function types that extend the ones
+defined in the specification. In those cases you can define a custom function type with for example:
+
+```json
+{
+"functions": [
+  {
+    "name": "sendOrderConfirmation",
+    "operation": "/path/to/my/script/order.ts#myOperation",
```

Comment:
maybe here instead of "myOperation" should use "myFunction" im not typescript expert tho :)